contracts/voting_escrow/src/contract.rs

```diff
@@ -386,7 +385,7 @@ fn checkpoint(
             if end > point.end && add_amount.is_zero() {
                 // This is extend_lock_time. Recalculating user's voting power
                 let mut lock = LOCKED.load(deps.storage, addr.clone())?;
-                let mut new_voting_power = calc_coefficient(dt).checked_mul(lock.amount)?;
+                let mut new_voting_power = lock.amount * calc_coefficient(dt);
```

Comment:
We need to keep checked behaviour. Use [this](https://github.com/astroport-fi/astroport-core/blob/e45685ce20808ddd14e5358a55d9d26128d77744/packages/astroport/src/lib.rs#L47) as an example.

Reply:
Ok, changed, thanks